Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -75,6 +77,22 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
         tableView.delegate = nil
     }
 
+    override func updateViewConstraints() {
+        tableView.snp_updateConstraints { make in
+            make.height.lessThanOrEqualTo(UIScreen.mainScreen().bounds.size.height).priorityHigh()
+        }
+        super.updateViewConstraints()
+    }
+
+    override func traitCollectionDidChange(previousTraitCollection: UITraitCollection?) {
```

Comment:
maybe `viewWillTransitionToSize` would be a better fit? Because then you can directly check stuff like  ` if UIDevice.current.orientation.isLandscape {...` instead of having the more complicated conditional